NSS-Day-Cohort-55/tabloidfullstack-ageless-pivotal-impalas
# Ls 39 view all categories

**DESC**
Added a 'Categories' tab to the NavBar, which will display a list of all Categories, in alphabetical order. Also, did some styling for the NavBar to add the logo, highlight the tab you're on and generally make it look more like it does in the mock-ups. 

ALSO I added SASS, so you'll need to npm install that.
**TESTING INSTRUCTIONS**
Categories should only be visible in the Nav when logged in
When Categories is selected it should become white in the Nav, when you return to home it should go back to gray
A list of all categories should display in alphabetical order, with (non-functioning) edit and delete buttons

Closes #39
